src/app/pages/EarnTez/components/earn-operation-modal.tsx

```diff
@@ -87,8 +87,8 @@ export const EarnOperationModal = <D, R extends TezosEarnReviewDataBase>({
       title={isInputDataStep ? inputDataStepTitle : confirmStepTitle}
       opened
       suspenseLoader={SuspenseLoader ? <SuspenseLoader isInputDataStep={isInputDataStep} /> : undefined}
-      titleRight={isInputDataStep ? undefined : null}
-      titleLeft={isInputDataStep ? null : <BackButton onClick={goToInputData} />}
+      titleRight={isInputDataStep ? undefined : <div />}
+      onGoBack={isInputDataStep ? undefined : goToInputData}
```

Comment:
is this a correct logic? `onGoBack` won't be use if `titleRight` is defined

Reply:
Yes, it is. If `onGoBack` is defined but `titleLeft` is not, then the default 'Back' button is added